modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/SourceAdapter.java

```diff
@@ -307,7 +308,8 @@ private void rollback() {
     @Override
     public Map<String, Object> createResponseParameters(CoreEvent event) throws MessagingException {
       try {
-        ResolverSetResult parameters = SourceAdapter.this.successCallbackParameters.resolve(from(event, configurationInstance));
+        ResolverSetResult parameters =
+            SourceAdapter.this.successCallbackParameters.resolve(from(event, configurationInstance, false));
```

Comment:
aren't you changing behavior here? The previous logic was equivalent to using `true`. The connectors expect the resolverset to have cursors not providers. 

Did you run all the SDK and http tests?

Reply:
Actually, this change in behavior is needed. Tests run ok, so it doesn't break anything and it fixes the source policies issue.

Reply:
Actually, this change in behavior is needed. Tests run ok, so it doesn't break anything and it fixes the source policies issue.